freqtrade/constants.py

```diff
@@ -380,6 +380,7 @@
         },
         'position_adjustment_enable': {'type': 'boolean'},
         'max_entry_position_adjustment': {'type': ['integer', 'number'], 'minimum': -1},
+        'backtest_signal_candle_export_enable': {'type': 'boolean'},
```

Comment:
while reading through the documentation, i did notice that this has not really good usability.

It's based on the assumption that we set this for every backtest (hence via config) - which is most likely wrong / unnecessary (you'll potentially test 20 things before looking at this in detail).

i'd rather combine this with the export flag  - so we can provide `--export signals` - which will then export both trades and signals ... that would allow this to be provided via cli argument - which should simplify only occasional use

Reply:
Sounds great! I'll take a look and see how that works and implement it

Reply:
well in the end, you just have to search for where `'export' == trades` is used via config (should only be backtesting, and obviously the constants). You'll then have to "augment" that to check both scenarios - as i'd assume that `--export signals` exports both (exporting only signals without trade does not make sense).

Reply:
Yup - looking through it now